coverage/historical/github/historical_coverage_report.py

```diff
@@ -401,6 +409,14 @@ def get_pull_requests(token, workspace, repo_name):
         nodes += result['data']['repository']['pullRequests']['nodes']
         has_next_page = result['data']['repository']['pullRequests']['pageInfo']['hasNextPage']
         end_cursor = result['data']['repository']['pullRequests']['pageInfo']['endCursor']
+    for node in nodes:
+        comments_authors = node['comments_authors']['nodes']
+        for comment_author in comments_authors:
+            if comment_author['author']['login'] in ['vibinex-dpu', 'vibinex-code-review']:
```

Comment:
These are the only two handles that I noticed. If there are any others, please let me know.